### ISSUE TITLE
README is missing import torch

### ISSUE THREAD

**Issue by londumas** [Lyken17/pytorch-OpCounter#104]
import torch is missing from readme. This is a small change that allows a simple
copy paste to run

```
import torch
from torchvision.models import resnet50
from thop import profile
model = resnet50()
input = torch.randn(1, 3, 224, 224)
macs, params = profile(model, inputs=(input, ))
```

**Comment by zhangxiann** [comment]
You are right

**Comment by sunhufei** [comment]
where is loss function?

**Comment by sunhufei** [comment]
Thank you?
